recipes/glib/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set major_minor = "2.48" %}
+{% set version = major_minor + ".0" %}
+{% set sha256 = "744be6931ca914c68af98dc38ff6b0cf8381d65e335060faddfbf04c17147c34" %}
+
+package:
+  name: glib
+  version: {{ version }}
+
+source:
+  fn: glib-{{ version }}.tar.xz
+  url: http://ftp.acc.umu.se/pub/GNOME/sources/glib/{{ major_minor }}/glib-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  # patches forked from https://github.com/pkgw/conda-recipes
+  #  Copyright 2014-2015 Peter Williams and collaborators.
+  # This file is licensed under a 3-clause BSD license.
+  patches:
+    - system-data-dirs.patch
+    - no-cocoa-notification.patch  # [osx]
+#    - find-libintl.h.patch  # [osx]
```

Comment:
Good to see we were able to build without this. The other patches are all basically the same as Homebrew patches. We should just replace them with Homebrew's in the feedstock.